src/Sylius/Bundle/OrderBundle/Form/DataMapper/OrderItemQuantityDataMapper.php

```diff
@@ -52,10 +52,15 @@ public function mapDataToForms($data, $forms)
      */
     public function mapFormsToData($forms, &$data)
     {
+        if (empty($data)) {
+            // Avoid a fatal error on trying to map data to null below.
+            // Not 100% sure why/how this happens so will see what this change does... PW 12/2016
+            return;
+        }
```

Comment:
I already made this change (but not the one below).

Try making your PR against `stable` instead.